### DISCUSSION TITLE
Alignment problems

### DISCUSSION THREAD

**Discussion by sky-lynx** [community/community#142069] [Programming Help] [answered]
### Body

I am making a pokedex website and trying to implement a search feature. However if I type at least 2 characters into the bar, everything on the page shifts to the right a little. Help is appreciated

Website link: https://sky-lynx.github.io/pokedex-api/
Code link: https://github.com/sky-lynx/pokedex-api

### Guidelines

- [x] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by davevad93** [comment, marked as answer]
Hi @sky-lynx ,

The issue is due to the layout shift caused by the scrollbar, to avoid this you can add the following line in your `css` inside the `body` tag:

```css
body {
    padding-left: calc(100vw - 100%);
}
```

**Reply by ebndev** [reply]
Hi @davevad93, as always- thank you for taking the time to answer! ❤️ 

@sky-lynx good luck with your projects! I see you have other Pokemon ones as well - always a good choice 😄

**Reply by davevad93** [reply]
Hi @ebndev , always glad to help around the Community! ❤️

**Comment by HimashaKodikara** [comment]
try to use tailwind css
